app/storage/metadata_parser.py

```diff
@@ -132,7 +134,7 @@ def convert_schema_name(self, data, **kwargs):
 
 
 def validate_questionnaire_claims(claims, questionnaire_specific_metadata):
-    """ Validate any survey specific claims required for a questionnaire"""
+    """Validate any survey specific claims required for a questionnaire"""
```

Comment:
These changes look unrelated to the pull request and should be removed.

Reply:
Done